src/frontend/app.ts

```diff
@@ -102,20 +102,36 @@ export default (octokitApp: App): Express => {
 
 	// Handlebars helpers
 	hbs.registerHelper("toLowerCase", (str) => str.toLowerCase());
+
 	hbs.registerHelper("replaceSpaceWithHyphen", (str) => str.replace(/ /g, "-"));
+
 	hbs.registerHelper(
 		"ifAllReposSynced",
 		(numberOfSyncedRepos, totalNumberOfRepos) =>
 			numberOfSyncedRepos === totalNumberOfRepos
 				? totalNumberOfRepos
 				: `${numberOfSyncedRepos} / ${totalNumberOfRepos}`
 	);
+
+	hbs.registerHelper("repoAccessType", (repository_selection) =>
+		repository_selection === "all" ? "All repos" : "Only select repos"
```

Comment:
Do we mean only `selected` repos here?

Reply:
This is from Pat's design spec:

<img width="911" alt="Screen Shot 2021-07-30 at 1 09 38 pm" src="https://user-images.githubusercontent.com/37155488/127594110-4b0e5aa2-06f5-4b2a-96d1-6f1aac2ff3b0.png">

which correlates with the terminology used by GH in the app settings:

<img width="242" alt="Screen Shot 2021-07-30 at 1 11 05 pm" src="https://user-images.githubusercontent.com/37155488/127594204-8ab8a73b-0025-4a12-93fe-e2a8f38612ca.png">

Reply:
I think this terminology is a bit confusing but it makes sense to be the same as GH.